app/screens/ServerView.tsx

```diff
@@ -50,20 +52,30 @@ const ServerView: React.FC<{
   const createServer = useCallback<() => Server>(() => {
     const salt = server?.salt || uuidv4()
     let token: string
+
+    // WHY:
+    // Some servers might still use a Subsonic API older than 1.13.0 and therefore need the token to be sent in plain text (e.g. the Music app for nextcloud).
+    // Also see http://www.subsonic.org/pages/api.jsp
+    // Use existing token (can be either hashed + salted or plain text)
     if (password === 'password' && server?.token) {
       token = server.token
+    } else if (usePlainPassword) {
+      // save password in plain text
+      token = password
```

Comment:
This actually causes a bit of an issue for people who have already entered their server details and then toggle the switch, it will save over their saved token.  If they are using the token method, I think we should clear the password field when this is enabled so that they must re-enter it, since they need to do that anyway and if they don't it won't work.  If they change from using plain text to not, then that's fine, we can just make the salt/token out of the plain text password.